packages/cfd/src/Constants/jurisdiction-bvi-contents.ts

```diff
@@ -0,0 +1,90 @@
+import { localize } from '@deriv/translations';
+import { TJurisdictionCardItems } from 'Components/props.types';
+
+export const jurisdiction_bvi_contents: TJurisdictionCardItems = {
+    is_over_header_available: false,
+    header: localize('British Virgin Islands'),
+    synthetic_contents: [
+        {
+            key: 'assets',
+            title: `${localize('Assets')}`,
+            description: `${localize('Synthetics, Basket indices and Derived FX')}`,
+            title_indicators: {
+                type: 'displayText',
+                display_text: `${localize('40+')}`,
+                display_text_skin_color: 'red-darker',
+            },
```

Comment:
change this to a function since localization doesnt work for objects when language is changed.